bin/test_pyodide.mjs

```diff
@@ -0,0 +1,13 @@
+import pyodide_pkg from 'pyodide/pyodide.js'
+
+const pyodide = await pyodide_pkg.loadPyodide()
+await pyodide.loadPackage([
+    'https://files.pythonhosted.org/packages/d4/cf/3965bddbb4f1a61c49aacae0e78fd1fe36b5dc36c797b31f30cf07dcbbb7/mpmath-1.2.1-py3-none-any.whl',  // latest mpmath on PyPI
+    'https://cdn.jsdelivr.net/pyodide/v0.20.0/full/numpy-1.22.3-cp310-cp310-emscripten_wasm32.whl',  // latest numpy built by pyodide
+    'http://localhost:8000/dist/sympy-1.11.dev0-py3-none-any.whl'  // git version sympy
```

Comment:
I think we can do it for mpmath and sympy. But numpy is dependent on pyodide version, which should be better changed when we upgrade pyodide version.
Currently pyodide version has to be hard coded because the only usable npm package of pyodide is alpha (not stable) version. Related: https://github.com/pyodide/pyodide/issues/2403

Reply:
It would also be nice if we don't have to update the version number in this file on each release.

Reply:
If we can make some of them automatically use newer versions then that's better. Anything pinned here is just work for later.

Reply:
Pyodide's npm support is not stable and doesn't contain any 3rd party python packages. I changed to use their contain-everything GitHub release file. Now only the version of pyodide itself is hard coded at runtests.yml.

Reply:
I looked at the [docs](https://pyodide.org/en/stable/usage/loading-packages.html) but I'm not entirely clear what the difference is between `loadPackage` and `micropip`. Is there a good reason to use one instead of the other?

Reply:
If we want to use `micropip`, we have to use `loadPackage` to load `micropip` first, then why not use `loadPackage` to load all dependencies?

Reply:
I don't know. That page makes it sound like micropip installs from PyPI whereas loadPackage loads from some other pyodide specific repository. Is that a correct reading? If so, does that mean that there is a lag between when these packages get updated on PyPI and when they get added to the pyodide repositories?

Reply:
Both are capable of loading package from a full URL. On my practice, the difference is how they handle a package name (not URL):
When `loadPackage` sees a package name, it only searches pyodide specific repository. The repo is determined by the location of `pyodide.js`, and can be overridden by `loadPyodide({indexURL: ...})`. If the repo doesn't contain the package, it throws.
When `micropip` sees a package name, it searches pyodide specific repo, and if not found it tries to search PyPI. Moreover, you can assign `sympy==1.8` when using `micropip`, and it will search PyPI for this specific version.


> there is a lag

Yes. It's not a big deal currently, because
* We don't rely on latest `mpmath` version. If we use `micropip`, we have to hard code mpmath version (because if we don't it will use the pyodide-hosted version). Pyodide is considering not hosting any pure-python packages, see https://github.com/pyodide/pyodide/issues/2580, but no decision is made. Pyodide itself even hasn't released 1.0 version, so we don't need to make everything perfect at this stage.
* `numpy` is maintained by pyodide, so there's no lag for it.
* `sympy` is git version.